Added release announcement - Stately Changelog 1 (#77)
statelyai/eng-blog: content/posts/2022-03-23-stately-changelog-1.mdx

---
title: "Stately Changelog #1 - Snap To Elements"

> description: The Stately Editor changelog, where we discuss the new things being released to the Stately Editor beta.

Reviewer: ```suggestion
description: The Stately Editor changelog is where we discuss new features released to the Stately Editor beta.
```

tags:
  - stately
  - announcement
  - editor

>   - beta

Reviewer: ```suggestion
  - beta
  - changelog
```

Add `changelog` to make it easy to group these changelog posts in the future.

author:
  - Matt Pocock
originalURL: ""
excerpt: ""
publishedAt: "2022-03-23"
---

> Happy Wednesday! Time for our first Editor Changelog blog, where we'll talk about the new updates we've shipped in the editor.

Reviewer: ```suggestion
Happy Wednesday! Time for our first Editor Changelog blog, where we’ll talk about the new updates we’ve shipped in the editor.
```

## Snap To Elements

> When you're placing states and events in the canvas, you'll now be able to snap them to surrounding elements.

Reviewer: ```suggestion
When you’re placing states and events in the canvas, you’ll now be able to snap them to surrounding elements.
```

> This is a hugely requested feature and should help make your charts neater and more legible.

Reviewer: ```suggestion
Snapping is a hugely requested feature and should help make your charts neater and more legible.
```
